odd_collector/adapters/redash/client.py

```diff
@@ -0,0 +1,121 @@
+from aiohttp import ClientSession
+from asyncio import gather
+from urllib.parse import urlparse
+from typing import NamedTuple, Tuple, Optional, Dict, Any, List
+from odd_collector.domain.plugin import RedashPlugin
+from .domain.query import Query
+from .domain.datasource import DataSource
+from .domain.dashboard import Dashboard
+
+
+class RequestArgs(NamedTuple):
```

Comment:
You have already had `RequestArgs` in `superset` adapter.
Move it to some helper/model module for reusing